data/json/terrain.json

```diff
@@ -7440,6 +7459,7 @@
     "type": "terrain",
     "id": "t_centrifuge",
     "name": "centrifuge",
+    "description": "This is centrifuge, a liquid separating device with an automated analyzer unit. It could be used to analyze a medical fluid sample, such as blood, if a test tube was placed in it.",
```

Comment:
```suggestion
    "description": "This is a centrifuge; a liquid separating device with an automated analyzer unit. It could be used to analyze a medical fluid sample, such as blood, if a test tube was placed in it.",
```